ion/textwriter_test.go

```diff
@@ -49,9 +49,9 @@ func TestWriteTextEmptyStruct(t *testing.T) {
 
 func TestWriteTextAnnotatedStruct(t *testing.T) {
 	testTextWriter(t, "foo::$bar::'.baz'::{}", func(w Writer) {
-		w.Annotation("foo")
-		w.Annotation("$bar")
-		w.Annotation(".baz")
+		w.Annotation(SymbolToken{Text: newString("foo"), LocalSID: SymbolIDUnknown})
```

Comment:
Up to you,  but if you think it makes it more readable you can use helper method for these like the suggestion on binarywriter_test